app/gateway/changelog.md

```diff
@@ -6,6 +6,282 @@ no_version: true
+
+* [**OpenTelemetry**](/hub/kong-inc/opentelemetry/) (`opentelemetry`)
+  * This plugin can now be scoped to individual services, routes, and consumers.
+  [#10096](https://github.com/Kong/kong/pull/10096)
+
+* [**StatsD**](/hub/kong-inc/statsd/) (`statsd`)
+  * Added the `tag_style` configuration parameter, which allows the plugin 
+  to send metrics with [tags](https://github.com/prometheus/statsd_exporter#tagging-extensions).
+  The parameter defaults to `nil`, which means that no tags are added to the metrics.
+  [#10118](https://github.com/Kong/kong/pull/10118)
+  
+* [**Session**](/hub/kong-inc/session/) (`session`), [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`), and [**SAML**](/hub/kong-inc/saml) (`saml`).
+
+  * These plugins now use `lua-resty-session` v4.0.0. 
+  
+    With this library update, these plugin have new session functionalities, including: configuring audiences to manage multiple sessions in a single cookie, global timeout, and persistent cookies.
+    [#10199](https://github.com/Kong/kong/pull/10199)
+
+* [**GraphQL Rate Limiting Advanced**](/hub/kong-inc/graphql-rate-limiting-advanced/) (`graphql-rate-limiting-advanced`) and [**Rate Limiting Advanced**](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`)
+    * In hybrid and DB-less modes, these plugins now support `sync_rate = -1` with any strategy, including the default `cluster` strategy.
+
+* [**OPA**](/hub/kong-inc/opa/) (`opa`)
+    * This plugin can now handle custom messages from the OPA server.
+
+* [**Canary**](/hub/kong-inc/canary/) (`canary`)
+    * Added a default value for the `start` field in the canary plugin. 
+    If not set, the start time defaults to the current timestamp.
+    
+    
+* **Improved Plugin Documentation**
+    * Updated the plugin compatibility information for more clarity on [supported network protocols] (/hub/plugins/compatibility/#protocols) and on [entity scopes](/hub/plugins/compatibility/#scopes) 
+    * Revised docs for top plugins to include examples:
+      * [CORS](/hub/kong-inc/cors/)
+      * [File Log](/hub/kong-inc/file-log/)
+      * [HTTP Log](/hub/kong-inc/http-log/)
+      * [JWT Signer](/hub/kong-inc/jwt-signer/)
+      * [Key Auth](/hub/kong-inc/key-auth/)
+      * [OpenID Connect](/hub/kong-inc/openid-connect/)
+      * [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/)
+      * [SAML](/hub/kong-inc/saml/)
+       * [StatsD](/hub/kong-inc/statsd/)
+       * [StatsD advanced](/hub/kong-inc/statsd-advanced/)
+  
+
+### Fixes
+
+#### Core 
+
+* Added back PostgreSQL `FLOOR` function when calculating `ttl`, so the returned `ttl` is always a whole integer.
+  [#9960](https://github.com/Kong/kong/pull/9960)
+* Exposed PostreSQL connection pool configuration.
+  [#9603](https://github.com/Kong/kong/pull/9603)
+* **Nginx template**: Do not add the default charset to the `Content-Type` response header when upstream response doesn't contain it.
```

Comment:
This one doesn't read like the others, are you telling the reader to not do that?

Reply:
Missed this one, it's from the OSS changelog. Will rephrase.